### ISSUE TITLE
International Humor Committee for Pun Approval and Joke Acceptance

### ISSUE THREAD

**Issue by gbezyuk** [vuejs/vuejs.org#547]
I guess there should be more jokes in the guide. It's not hard to read as it is, but having a little fun never hurts.

**Comment by chrisvfritz** [comment]
Ha, it's not every day that someone asks for _more_ of my jokes, so thank you! 😂  I'm an educator and in my curricula focused on Americans, I do try to include a lot of humor and I've received feedback from students that it's very much appreciated by most. 

When writing the Vue docs however, I've tried to hold myself back more. The reason is Vue's community is much more international and I've learned from past experience that humor can be difficult to translate. Sarcasm can be offputting or easy-to-miss, puns usually require a stronger grasp of the language, and general silliness can come across as strange, childish, or even insulting to some.

That said, I'd be open to PRs to include more jokes (one joke at a time please). 🙂  Our community and core team are spread throughout the world, so we can reference new jokes in this issue and review them one at a time for their international appeal/acceptance.

**Comment by gbezyuk** [comment]
Sounds great. I believe, according to the online/hypertext nature of this documentation one of the best ways for jokes here could be trough citations of well-known comic figures. For one, nobody expects the Spanish Inquisition. Especially in list rendering examples.

**Comment by gbezyuk** [comment]
In the "Vue vs. Angular 2: Learning Curve" section of comparison.md there's a place looking suitable for some good old British Humour. I suggest replacing this sentence:

`It's an understatement to say that [Vue's Hello World](index.html#Hello-World) is considerably simpler. It's so trivial in fact, that we don't even dedicate a whole page in the guide to it.`

with something more humble, like:

`It wouldn't be an exaggeration to say that [Vue's Hello World](index.html#Hello-World) is a little bit simpler. Maybe because of that, we don't even need to dedicate a whole page in the guide to it.`

**Comment by chrisvfritz** [comment]
@gbezyuk I'm open to that PR. 🙂

**Comment by gbezyuk** [comment]
@chrisvfritz see #573

**Comment by gbezyuk** [comment]
A bit on the Spanish Inquisition mentioned above, or how to stay true to Python even while using JavaScript.
https://github.com/gbezyuk/minimalistic-yet-fully-featured-vuex-example/blob/standalone-vue/img/step1.png

**Comment by chrisvfritz** [comment]
@gbezyuk Ha, nice. I'd like to avoid pop culture references though - even ones that are well-known in many parts of the world.

**Comment by gbezyuk** [comment]
For a Russian translation only, in the description of v-show, I propose to use a reference to a famous post-soviet movie quote. As `v-show` does not really remove nodes from DOM tree, the well-known idiom "can you see the gopher? anyway, there is one..." feels to fit: 

## v-show

...

``` html
<div v-show="Math.random() > 0.5">
  Видишь суслика? А он есть...
</div>
```

**Comment by chrisvfritz** [comment]
@gbezyuk Sounds good to me! I think pop-culture references are OK in less global languages, as long as you believe ~90% of readers will understand the reference and it won't be more confusing for the other 10%. I'll leave that judgment to you. 🙂

**Comment by lionel-bijaoui** [comment]
> The reason is Vue's community is much more international and I've learned from past experience that humor can be difficult to translate. Sarcasm can be offputting or easy-to-miss, puns usually require a stronger grasp of the language, and general silliness can come across as strange, childish, or even insulting to some.

Yes! Thank you for that!
I'm not against fun, silly pun and all the jazz. But your jokes could also be dated, cringy, dad level bad, and/or just plain uncool. You think you meme are fresh but the guide will stay the same for years.
Please keep the guide a tool to learn about vuejs without any distraction. *Yeah, I'm fun at parties...*

I personally like to find little nuggets of humor here and there, but I know people who will not take these in a good way. It could give a bad (not "serious business") image to these people and Vuejs is still relatively small and don't need that.
You can have all the fun in the world in the chat/forum/subreddit/whatever.

To sum up:
- bad point
  - hard to translate
  - could be misinterpreted (seem strange, childish, unprofessional, insulting, dated)
  - could be misunderstood
  - create a distraction from the content of the guide
  - could be polarizing for the community
- good point
  - make some dev giggle or more (we need that sometime)
  - give a personality to the guide and therefore Vuejs as a whole
  - motivate people to read the guide

That was just my humble opinion. Sorry for being a party pooper...

**Comment by phanan** [comment]
I guess we can close this issue now and address its spirit in a case by case basic.
